lib/elsa/topic.ex

```diff
@@ -71,6 +71,7 @@ defmodule Elsa.Topic do
     end)
   end
 
+  @spec send_request(pid(), term(), integer()) :: :ok | {:ok, term()} | {:error, term()}
```

Comment:
do you want to add a spec to this private function?

Reply:
Nice catch. I think this was me debugging issues and was mistakenly left behind.